pages/Manuscripts.xml

```diff
@@ -17,76 +17,117 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+               </cit></p>
+            <p>This means that for manuscripts made of only one part we will use only
+                  <gi>msDesc</gi>, but in any case, where a structure in codicological units occurs,
+               we will use <gi>msPart</gi>. <gi>msPart</gi> can contain other <gi>msPart</gi>s.
+                  <gi>msFrag</gi> can be used when there are at least two parts of one manuscript
+               kept in different repositories and in our schema this is allowed also beside
+                  <gi>msPart</gi> for special cases like <ref target="BNFet45" type="bm"
+                  >BNFet45</ref>.</p>
+            <p>As for <ref target="https://github.com/BetaMasaheft/Documentation/issues/368">issue
+                  368</ref> we have decided to use <gi>msFrag</gi> and <gi>msPart</gi> as follows in
+               cases where leaves or quires have been moved. The Ms containing currently the unit
+               will have a <gi>msPart</gi> with it, with an appropriate description and pointer to
+               the manuscript where the unit comes from. The Ms which has lost a unit will instead
+               have a <gi>msFrag</gi> pointing to the Ms where the unit is now. Please, do not
+               confuse this with cases where leaves or quires are lost.</p>
          </div>
          <div type="level2">
             <head>Double records</head>
             <p>It can be that you find out the same manuscript is present under 2 identifiers.</p>
-            <p>You can fix this by merging the two records in one and deleting the one with less information. Remember when doing this to check for references to the id which will go missing, and update them to point to the other identifier.</p>
-            </div>
+            <p>You can fix this by merging the two records in one and deleting the one with less
+               information. Remember when doing this to check for references to the id which will go
+               missing, and update them to point to the other identifier.</p>
+         </div>
+         <div type="level2">
+            <head>Differenciating between various levels of description</head>
+            <p>To mark a rudimentary description as "stub" (if the cataloguer wants others consider
+               it as "stub"), one needs to write the word "stub" anywhere in the change history.
+                  <egXML xmlns="http://www.tei-c.org/ns/Examples">
+                  <revisionDesc>
+                     <change who="ES" when="2024-04-10">created stub</change>
+                  </revisionDesc></egXML> When the record has stopped being a stub as it has been
+               developed further then in the first change line the word stub must replaced with
+               something like "initial version" or anything else. After the description is
+               completed, the above example can be changed into something like the example below.
```

Comment:
I would remove the sentence

"After the description is
               completed, the above example can be changed into something like the example below."


It repeats what has been said and misleads, as this change is necessary not only when the description has been **completed**, but at any point when it is not a stub any more (the judgement on that is up to the cataloguer of course)